enterprise_subsidy/apps/api/v1/serializers.py

```diff
@@ -106,6 +109,67 @@ def get_unit(self, obj):
         return obj.ledger.unit if obj.ledger else None
 
 
+class TransactionCreationError(Exception):
+    """
+    Generic exception related to errors during transaction creation.
+    """
+
+
+class CreateTransactionSerializer(serializers.ModelSerializer):
```

Comment:
My favorite takeaway from the enterprise-access code was finding either "Request" or "Response" in the name of the serializer, which really helped me understand what the serializer was actually used for.  Is that a general pattern you like, and should we adopt it here by calling it CreateTransactionRequestSerializer?